website/docs/r/container_app.html.markdown

```diff
@@ -335,7 +335,7 @@ An `ingress` block supports the following:
 
 * `fqdn` -  The FQDN of the ingress.
 
-* `external_enabled` - (Optional) Is this an external Ingress.
+* `external_enabled` - (Optional) Is this an external Ingress? If `internalOnly` is true in the Container Apps Environment, external ingress is limited to the VNet.
```

Comment:
We need to be clear that this just means exposing connectivity to the container via this ingress outside the Container App Environment, so I think this should read:
```suggestion
* `external_enabled` - (Optional) Are connections to this Ingress from outside the Container App Environment enabled? Defaults to `false`.
```

Reply:
I'm happy with that, LGTM :)